README.md

```diff
@@ -39,7 +47,11 @@ We welcome contributions and suggestions! Please see the [contributing guideline
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). Please see the [code of conduct](CODE_OF_CONDUCT.md) for details.
 
-## Reference
- 
-- [Documentation](https://docs.microsoft.com/azure/machine-learning)
- 
+[tutorials]: ./tutorials
```

Comment:
The R comment could be put here

e.g.

[azureml R cli extension examples](.cli/jobs/single-step/r)